auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -1,19 +1,47 @@
 <template>
   <section id="testimonials-container">
     <v-container class="quote-container">
-      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
-        and the role we want to play."</p>
-      <p><strong>&ndash; BC Benefit Company user</strong></p>
+      <div :class="{ 'fadeOut': isFading, 'fadeIn': !isFading}">
+        <p>{{quotes[quoteIndex].text}}</p>
+        <p><strong>&ndash; {{quotes[quoteIndex].author}}</strong></p>
+      </div>
     </v-container>
   </section>
 </template>
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
-import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 
 @Component({})
-export default class TestimonialQuotes extends Vue {}
+export default class TestimonialQuotes extends Vue {
+  private isFading: boolean = false
+  private quoteIndex: number = 0
+  private quotes: Array<any> =
+    [
+      {
+        text: 'We\'re incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values and the role we want to play.',
+        author: 'BC Benefit Company user'
+      },
+      {
+        text: 'This is a placeholder for a second quote.',
+        author: 'BC Benefit Company user'
+      },
+      {
+        text: 'This is a placeholder for third quote.',
+        author: 'BC Benefit Company user'
+      }
+    ]
+
+  mounted () {
+    setInterval(() => {
```

Comment:
hm..can this be done in better ways ? may be using transitions..

Reply:
Looking into Transitions!

Reply:
FYI, browser transitions can use hardware acceleration, so they don't processing power away from your code!

Reply:
Simplified this a bit, let me know what you think! I didn't find transitions all that intuitive, actually found base CSS more preferable, though always open to ideas.

Reply:
I'm OK with CSS transitions.